server.js

```diff
@@ -1,34 +0,0 @@
-///---------------- imports 
-const express = require('express')
-const app = express()
-const path = require('path')
-const fs = require('fs');
-const port = process.env.PORT || 8000
-
-//---------------middleware
-app.use(express.static (path.resolve('./public')))
-
-app.get('/api', (request, res) => {
-    res.sendFile(path.resolve('./API/restaurant-directory.json'))
-  });
-
-app.get('/api/:restaurant', (request, response) => {
-    response.sendFile(path.resolve(`./API/${params.path.restaurant}`)) // 
-    // LOOK UP HOW TO ACCESS PARAMS
-    //You need to change your variable on line 15 to access the params.restauramnt
-//But there's a syntaxes to it that im not familiar with off the top of my head so you'll have to look it up
- 
-});
```

Comment:
the syntax that you are using to access your params is off. I linked an article in the overall review to information about accessing params. changing that up can help out with what you're trying to accomplish here